flashinfer/fused_moe/cute_dsl/w4a8_mxfp4_moe.py

```
@@ -0,0 +1,274 @@
+from .moe_reduce_triton import moe_reduce, build_reduce_index
+
+try:
+    import cutlass
+
+    _ACC = cutlass.Float32
+    _CUTLASS_DTYPE = {
+        torch.float16: cutlass.Float16,
+        torch.bfloat16: cutlass.BFloat16,
+        torch.float32: cutlass.Float32,
+    }
+except Exception:  # pragma: no cover
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)